### ISSUE TITLE
Add an option to include spacing (px) between indicators

### ISSUE THREAD

**Issue by fossfreedom** [UbuntuBudgie/budgie-indicator-applet#24]
https://discourse.ubuntubudgie.org/t/appindicator-applet-feature-request/4102

So in budgie desktop settings have an option to define the px between indicators

**Comment by fossfreedom** [comment]
Ok - no need to have a specific setting in budgie-indicator-applet - in budgie 10.7 a panel will have a spacing value defined.

So now need a method get that spacing value and apply it.

**Comment by Noblepal** [comment]
> Ok - no need to have a specific setting in budgie-indicator-applet - in budgie 10.7 a panel will have a spacing value defined.
> 
> So now need a method get that spacing value and apply it.

Yeah, would be really nice to have budgie-indicator-applet matching the panel's icon spacing.

![image](https://user-images.githubusercontent.com/20543843/165047040-1630f37d-921d-4241-93c3-77b2b608aa09.png)
